website/.env.example

```diff
@@ -1,11 +1,11 @@
 APP_ENV=local
 APP_DEBUG=true
-APP_KEY=SomeRandomString
+APP_KEY=Y57RWgDoeEuNLlltuBXk6q0BMLYtjrQy
```

Comment:
Uh oh, you just made your app key and db credentials public...

Don't change the .example files.